common/ucf-faq-list-common.php

```diff
@@ -211,7 +211,9 @@ public static function get_related_faqs_by_topic( $topics, $excluded_faqs ) {
 
 			foreach( $topics as $topic ) {
 				$related = get_field( 'related_faqs', "topic_{$topic->term_id}" );
-				$retval = array_merge( $retval, $related );
+				if( $related ) {
```

Comment:
You're probably going to want to make sure $related is an array before trying to merge it with another array as well. `is_array` will do the trick.